app/main.py

```diff
@@ -25,12 +28,15 @@ def serve_cars(self, cars: list) -> float:
         return round(income, 1)
 
     def calculate_washing_price(self, car: Car) -> float:
-        cost = car.comfort_class * (self.clean_power - car.clean_mark) * \
-            self.average_rating / self.distance_from_city_center
+        cost = car.comfort_class\
+            * (self.clean_power - car.clean_mark)\
+            * self.average_rating\
+            / self.distance_from_city_center
         return round(cost, 1)
```

Comment:
In general, you don't need to use the `\` symbol, if you take the expression in parentheses `()`, it was in the mentor's remark. You can also avoid creating an additional variable and return the result at once